content/docs/command-reference/repro.md

```diff
@@ -51,7 +51,7 @@ There are a few ways to restrict what will be regenerated by this command: by
 specifying specific reproduction [`targets`](#options), or by using certain
 command [options](#options), such as `--single-item` or `--all-pipelines`.
 
-> Note that stages without dependencies or outputs are considered [always
+> Note that stages without dependencies or outputs are considered [always >
 > changed], so `dvc repro` always executes them.
```

Comment:
This looks like an error in Prettier. It was also happening locally so I force committed this without the trailing `>`.

Reply:
As we can see in https://dvc-org-ref-states-epk8gmpjlt3.herokuapp.com/doc/command-reference/repro this renders just fine as-is so I'm going to ignore this restyled PR.

<img width="722" alt="image" src="https://user-images.githubusercontent.com/1477535/154164867-74408f2c-0819-44b3-956c-73ecd3ac95c6.png">